airgun/views/subscription.py

```diff
@@ -89,13 +89,25 @@ class SubscriptionListView(BaseLoggedInView, SubscriptionSearchableViewMixin):
             'Name': Text("./a"),
         }
     )
+
+    scc_button = Text("//i[contains(@class,'fa fa-columns tooltip-button')]")
     add_button = Button(href='subscriptions/add')
     manage_manifest_button = Button('Manage Manifest')
     export_csv_button = Button('Export CSV')
     delete_button = Button('Delete')
     progressbar = ProgressBar('//div[contains(@class,"progress-bar-striped")]')
     confirm_deletion = DeleteSubscriptionConfirmationDialog()
 
+    name = Checkbox(id='id')
+    sku = Checkbox(id='product_id')
+    contract = Checkbox(id='contract_number')
+    start_date = Checkbox(id='start_date')
+    end_date = Checkbox(id='end_date')
+    req_virt_who = Checkbox(id='virt_who')
+    type = Checkbox(id='type')
```

Comment:
`type` is a reserved name, of course it is not important, but please rename that variable